QuickFIXn/Session.cs

```diff
@@ -348,14 +349,22 @@ public virtual bool Send(Message message)
         /// Sends a message
         /// </summary>
         /// <param name="message"></param>
+        /// <param name="messageType"></param>
         /// <returns></returns>
-        public bool Send(string message)
+        public bool Send(string message, string messageType)
```

Comment:
breaking change?

Reply:
Wasn't supposed to be - it's been reverted. What do you think of what it is now together with #909?